tests/test_kwargs_and_defaults.cpp

```diff
@@ -37,17 +39,21 @@ TEST_SUBMODULE(kwargs_and_defaults, m) {
     m.def("args_function", [](py::args args) -> py::tuple {
```

Comment:
This can probably be safely changed, I just didn't know if you were testing fo rpass by value explicitly.